app/api/v1/endpoints/admin.py

```
@@ -588,6 +653,232 @@ async def list_users(
         )
 
 
+@router.post("/users/verify/{user_id}")
+async def verify_user(
+    user_id: str,
+    current_user: User = Depends(require_role([UserRole.ADMIN])),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)